### DISCUSSION TITLE
Time to sync full node

### DISCUSSION THREAD

**Discussion by antoniocarras** [Chia-Network/chia-blockchain#1648] [Q&A]
I have two questions about full node time to sync.
1. I am experiencing long times to sync the full node. Which is approximately expected time to sync the full node? It takes me 24 hours...
2. Each time I start the chia blockchain GUI it starts syncing from the beginning. Is it not posible to sync since the last block sync in previous sessions?
Thanks in advance

**Comment by mariano54** [comment]
It should start from the point at which it left off.

Yes 24 hours is normal

**Reply by antoniocarras** [reply]
Thanks very much for your quick answer.
I am using Ubuntu 20.1 and it always starts from block 0. Any advice where to look for finding the cause of this problem?
